### PR TITLE
Added support for CSS style sheets in SVGs

### PR DESCRIPTION
**Pull request by claudep**
Use tinycss2 and cssselect2 to read and apply style sheets.
Refs #111.

### CODE REVIEW

**Comment by claudep** [comment]
This may need a bit more real-world testing before integration in master.

**Comment by deeplook** [comment]
I can only check this very superficially. It looks ok to me, but it would be nice, maybe, to have just a bit more test cases, and feedback from other users. Can we identify any sample files inside the official SVG test-suite we're running with CSS that work better than before? 

And for a release like 0.9.0 we should then at least mention the feature in the current README, so people know CSS support is there in principle. And then we have something to expand later, ideally in some real, Sphinx-based documentation, say for 1.0, and on readthedocs.io.

**Comment by claudep** [comment]
I may have a little time to spend on it tomorrow.

**Comment by claudep** [comment]
I didn't find any stylesheet usage in the official SVG test suite.
I updated the limitation note in the README file. I'm afraid that feedback from users won't come until we release that code on PyPi.

File: README.rst [hunk 1]
```diff
@@ -64,7 +64,8 @@ Features
 Known limitations
 -----------------
 
-- stylesheets are not supported (only the style attribute)
+- experimental support for stylesheets was introduced in 0.9.0. Please report
```
Comment (deeplook) [inline README.rst]: I would skip any version number here until we get a release out.

**Comment by deeplook** [comment]
@claudep See my comment on the release number above. Otherwise fine, and let's merge it!

**Pull request merged**